Integrations/Gmail/Gmail.py

```diff
@@ -24,7 +32,7 @@
 SCOPES = ['https://www.googleapis.com/auth/admin.directory.user.readonly']
 PROXY = demisto.params().get('proxy')
 DISABLE_SSL = demisto.params().get('insecure', False)
-
+FETCH_TIME = demisto.params().get('fetch_time', '1 days')
```

Comment:
```suggestion
FETCH_TIME = demisto.params().get('fetch_time', '1')
```

Reply:
@yaakovi said I should keep the convention of using '1 days' in fetch time and the parse_date_range function